util/reflect.go

```diff
@@ -62,3 +62,84 @@ func MustWalkTerraformOutput(value interface{}, path ...string) interface{} {
 	}
 	return found
 }
+
```

Comment:
Would you consider using third party packages that cover more types? This way we wouldn't have problems when adding new types to `TerragruntConfig` in the future.

https://github.com/barkimedes/go-deepcopy
https://github.com/huandu/go-clone